src/com/dotcms/plugin/saml/v3/OpenSamlAuthenticationServiceImpl.java

```diff
@@ -189,6 +190,10 @@ private AttributesBean resolveAttributes (final Assertion assertion, final Confi
 
         final AttributesBean.Builder attrBuilder = new AttributesBean.Builder();
 
+        if (validateAttributes(assertion)){
+            throw new AttributesNotFoundException("No attributes found");
+        }
+
```

Comment:
Minor thing: let's have the validateAttributes throw the exception and replace these three lines with just one call to validateAttributes